lib/std/special/docs/main.js

```diff
@@ -521,9 +521,7 @@
     function getCallHtml(fnDecl, callIndex) {
         var callObj = zigAnalysis.calls[callIndex];
 
-        // TODO make these links work
-        //var html = '<a href="' + navLinkCall(callObj) + '">' + escapeHtml(fnDecl.name) + '</a>(';
-        var html = escapeHtml(fnDecl.name) + '(';
+        var html = '<a href="' + navLinkDecl(fnDecl) + '">' + escapeHtml(fnDecl.name) + '</a>(';
```

Comment:
This isn't correct, the commented code creates a correct link but it just leads to a 404 since we don't have data on generic instantiations yet #3407.

Reply:
Yes, this should have been reverted together with the other code I threw out. Thanks for taking a look!